app/models/concerns/hyrax/ability.rb

```diff
@@ -188,5 +193,15 @@ def user_is_depositor?(document_id)
       def curation_concerns_models
         [::FileSet, ::Collection] + Hyrax.config.curation_concerns
       end
+
+      def can_review_submissions?
+        # Are there any workflows where this user has the "approving" responsibility
+        approving_role = Sipity::Role.find_by(name: 'approving'.freeze)
+        return false unless approving_role
+        Hyrax::Workflow::PermissionQuery.scope_processing_agents_for(user: current_user).any? do |agent|
```

Comment:
@jeremyf I'm pushing a small change to this branch in hopes that we can get *something* merged in the short-term while we consider what you have suggested, which I wrote up here: https://github.com/projecthydra-labs/hyrax/issues/956